concepts.tex

```diff
@@ -13,17 +13,19 @@
 \pnum
 This Clause describes library components that \Cpp programs may use to perform
 compile-time validation of template parameters and perform function dispatch
-based on properties of types.
+based on properties of types. \newtxt{The purpose of these concepts is to establish
+a foundation for equational reasoning in programs.}
 
 \pnum
-The following subclauses describe core language concepts, foundational concepts,
-function concepts, iterator concepts, and rearrangement concepts
-as summarized in Table~\ref{tab:concepts.lib.summary}.
+The following subclauses describe core language concepts, \oldtxt{foundational concepts,}
+\newtxt{comparison concepts, object concepts,} function concepts, iterator concepts,
```

Comment:
The iterator concepts were in the Iterators chapter. I haven't finished reading yet, so maybe you moved them.